src/io.test.js

```diff
@@ -37,8 +36,8 @@ test("batch", async () => {
   ]);
 
   expect.assertions(2);
-  expect(result[0]).toEqual(result[1]);
-  expect(result[0]).toEqual(22);
+  expect(result[0]).toEqual(10);
```

Comment:
why did this change ?

Reply:
queue is being flushed on the first call to batch since the condition is true. Unlike previous behaviour where the function would have waited the `waitTime` anyway